ui/StatusQ/src/StatusQ/Components/StatusLetterIdenticon.qml

```diff
@@ -31,7 +31,7 @@ Rectangle {
         anchors.centerIn: parent
         width: Math.round(parent.width / 2)
         height: Math.round(parent.height / 2)
-        emojiId: Emoji.iconId(root.emoji, root.emojiSize) || ""
+        emojiId: Emoji.iconId(root.emoji, root.emojiSize) || Emoji.iconHex(root.emoji) || ""
```

Comment:
@jrainville I figured this way we can also preserve the backwards compatibility with things like the Emoji popup, e.g. when setting the channel emoji